langlib/lang.map/src/main/ballerina/errors.bal

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Shall we add the licence header here as well?

Reply:
added